Exercise 3/Js/.vscode/Es.js

```diff
@@ -0,0 +1,44 @@
+// change everything below to the newer Javascript!
+
+// let + const
+// Destructuring
+class person{
+  constructor( firstName, lastName, age, eyeColor, city ){
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.eyeColor = eyeColor;
+    this.city = city;
+  }
+}
+
+myperson = new person( "Mario", "Poire", 27, "Green", "mission" )
+
+// Object properties
+let a = 'test';
+let b = true;
+let c = 789;
+a = 'test2';
+
+const okObj = {
+  a: a,
+  b: b,
+  c: c
+};
+
+console.log(Object.keys(okObj));
```

Comment:
what are you trying to do here?

Reply:
I was just printing it out to make sure it came out it was more for me than for the assignment i should have commented it out.